File: 0283-move-zeroes/0283-move-zeroes.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int count=0;
        int n=nums.size();
        for(int i=0; i<n; i++){
            if(nums[i]!=0){
                int temp=nums[count];
                nums[count++]=nums[i];
                nums[i]=temp;
            }
        }
        
    }
};

File: 0283-move-zeroes/0283-move-zeroes.py
class Solution(object):
    def moveZeroes(self, nums):
        """
        :type nums: List[int]
        :rtype: None Do not return anything, modify nums in-place instead.
        """
            	# code here
    	count=0
        n=len(nums)
    	for i in range(0,n):
    	    if nums[i] is not 0:
    	        temp=nums[count]
    	        nums[count]=nums[i]
    	        nums[i]=temp
    	        count+=1